src/components/pagination/_pagination.scss

```diff
@@ -300,10 +301,11 @@ $css--helpers: true;
 
   span.#{$prefix}--pagination__text {
     margin-left: $carbon--spacing-05;
-    color: $text-02;
+    color: $text-01;
   }
-  .#{$prefix}--pagination__text[for='select-id-pagination-page'] {
-    margin-left: -1rem;
+
+  .#{$prefix}--pagination__right span.#{$prefix}--pagination__text {
```

Comment:
@joshblack you were totally right about renaming this selector in https://github.com/IBM/carbon-components/pull/2076#discussion_r265198893 ! i didn't realize the ID would be changing on this element. this selector change will apply to any `span.#{$prefix}--pagination__text` but for the moment (and the foreseeable future) there will just be the one `<select>` and label there!